bentoml/_internal/runner/runner.py

```diff
@@ -99,7 +100,7 @@ async def async_run(self, *args: t.Any, **kwargs: t.Any) -> t.Any:
 
 
 @attr.define
-class RunnerHandle:
+class RunnerHandle(ABC):
```

Comment:
lol, so you take the `RunnerImpl` back with another name. I write the current because my previous implementation (RunnerImpl) is disliked.

Reply:
Oh, why?